Create 2023-WasmCon.md (#710)
fermyon/developer: content/events/2023-WasmCon.md

title = "WasmCon 2023"
date = "2023-06-29T00:00:01Z"

> description = "Fermyon is super excited for the inaugural WasmCon event hosted by the Linux Foundation. Taking place in Bellevue, WA, this conference will include hands-on workshops and talk sessions. Be sure to be on the look out for ByteCode Alliance activites the day after as well."

Reviewer: ```suggestion
description = "Fermyon is super excited about the inaugural WasmCon event hosted by the Linux Foundation. Taking place in Bellevue, WA, this conference will include hands-on workshops and talk sessions. Be sure to be on the lookout for ByteCode Alliance activities the day after."
```
